docs/pages/guides/icons.md

```diff
@@ -2,227 +2,85 @@
+It follows the similar configuration as `createIconSetFromIcoMoon` as shown in the example:
 
-<SnackInline
-label='Icon images'
-files={{
-    'assets/images/slack-icon.png': 'https://snack-code-uploads.s3.us-west-1.amazonaws.com/~asset/0397c8d3e7445a4e826705f08abdd8ef'
-  }}>
+```javascript
+// Import the createIconSetFromFontello method
+import { createIconSetFromFontello } from '@expo/vector-icons';
 
-```jsx
-import * as React from 'react';
-import { Image, View, StyleSheet } from 'react-native';
-
-export default function App() {
-  return (
-    <View style={styles.container}>
-      <Image
-        source={require('./assets/images/slack-icon.png')}
-        fadeDuration={0}
-        style={{ width: 50, height: 50 }}
-      />
-    </View>
-  );
-}
+// Import the config file
+import fontelloConfig from './config.json';
 
-/* @hide const styles = StyleSheet.create({ ... }); */
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
-  },
-});
-/* @end */
+// Both the font name and files exported from Fontello are most likely called "fontello"
+const Icon = createIconSetFromFontello(fontelloConfig, 'fontello', 'fontello.ttf');
 ```
 
-</SnackInline>
-
-Let's assume that our `SlackIcon` class is located in `my-project/components/SlackIcon.js`, and our icon images are in `my-project/assets/images`, in order to refer to the image we use require and include the relative path. You can provide versions of your icon at various pixel densities and the appropriate image will be automatically used for you. In this example, we actually have `slack-icon@2x.png` and `slack-icon@3x.png`, so if I view this on an iPhone 6s the image I will see is `slack-icon@3x.png`. More on this in the [Images guide in the react-native documentation](https://reactnative.dev/docs/images#static-image-resources).
+## Icon images
 
-We also set the `fadeDuration` (an Android specific property) to `0` because we usually want the icon to appear immediately rather than fade in over several hundred milliseconds.
+You can use the `Image` component from React Native to display an icon. The `source` prop takes the relative path to refer the image.
 
-## Button Component
+<SnackEmbed snackId="@amanhimself/image-icons-expo-example" preview platform="web" />
 
-A convenience component for creating buttons with an icon on the left side.
+You can also provide different versions of your icon at various pixel densities. The `Image` component takes care of using the image with appropriate pixel density automatically. For example, if the image has variants like `icon@2x.png` and `icon@3x.png`, the `@2x` suffix is served for a device's screen density such as iPhone 7 and the `@3x` suffix is served for a device's screen density such as iPhone 7 Plus. [You can learn more about serving different densities in React Native documentation](https://reactnative.dev/docs/images#static-image-resources).
```

Comment:
Since the iPhone 7 is rather old, but there's also almost no phones that Apple makes anymore that would use @2, maybe illustrate this difference with old vs new, rather than small/ base vs big/ premium, e.g., "@2 is for iPhone 8 while the iPhone 13 would use @3"

Reply:
That's a great idea. I'll change this to reflect old vs new devices.